sdk/servicenetworking/arm-servicenetworking/CHANGELOG.md

```diff
@@ -1,15 +1,92 @@
+  - Added Enum KnownPolicyType
+  - Added Enum KnownVersions
+  - Enum KnownAssociationType has a new value subnets
+  - Enum KnownOrigin has a new value "user,system"
+  - Enum KnownOrigin has a new value system
+  - Enum KnownOrigin has a new value user
+  - Added function restorePoller
 
-### Other Changes
+### Breaking Changes
 
+  - Removed operation AssociationsInterface.beginCreateOrUpdate
+  - Removed operation AssociationsInterface.beginCreateOrUpdateAndWait
+  - Removed operation AssociationsInterface.beginDelete
+  - Removed operation AssociationsInterface.beginDeleteAndWait
+  - Removed operation FrontendsInterface.beginCreateOrUpdate
+  - Removed operation FrontendsInterface.beginCreateOrUpdateAndWait
+  - Removed operation FrontendsInterface.beginDelete
+  - Removed operation FrontendsInterface.beginDeleteAndWait
+  - Removed operation TrafficControllerInterface.beginCreateOrUpdate
+  - Removed operation TrafficControllerInterface.beginCreateOrUpdateAndWait
+  - Removed operation TrafficControllerInterface.beginDelete
+  - Removed operation TrafficControllerInterface.beginDeleteAndWait
+  - Class ServiceNetworkingManagementClient has a new signature
+  - Interface AssociationsInterfaceCreateOrUpdateOptionalParams no longer has parameter resumeFrom
+  - Interface AssociationsInterfaceDeleteOptionalParams no longer has parameter resumeFrom
+  - Interface FrontendsInterfaceCreateOrUpdateOptionalParams no longer has parameter resumeFrom
+  - Interface FrontendsInterfaceDeleteOptionalParams no longer has parameter resumeFrom
+  - Interface ServiceNetworkingManagementClientOptionalParams no longer has parameter $host
+  - Interface ServiceNetworkingManagementClientOptionalParams no longer has parameter endpoint
+  - Interface TrafficControllerInterfaceCreateOrUpdateOptionalParams no longer has parameter resumeFrom
+  - Interface TrafficControllerInterfaceDeleteOptionalParams no longer has parameter resumeFrom
+  - Parameter id of interface AssociationSubnetUpdate is now required
+  - Type of parameter tags of interface AssociationUpdate is changed from {
+        [propertyName: string]: string;
+    } to Record<string, string>
+  - Type of parameter info of interface ErrorAdditionalInfo is changed from Record<string, unknown> to Record<string, any>
+  - Type of parameter tags of interface FrontendUpdate is changed from {
+        [propertyName: string]: string;
+    } to Record<string, string>
+  - Type of parameter tags of interface TrackedResource is changed from {
+        [propertyName: string]: string;
+    } to Record<string, string>
+  - Type of parameter tags of interface TrafficControllerUpdate is changed from {
+        [propertyName: string]: string;
+    } to Record<string, string>
+  - Class ServiceNetworkingManagementClient no longer has parameter $host
+  - Class ServiceNetworkingManagementClient no longer has parameter apiVersion
+  - Class ServiceNetworkingManagementClient no longer has parameter subscriptionId
```

Comment:
why we have these two breakings?

Reply:
these two are from hlc -> modular

Reply:
Have we reviewed the breaking? If no could you create an issue to track that?

Reply:
Issue created: https://github.com/Azure/autorest.typescript/issues/3049

Reply:
Accepted this breakings.